src/item.cpp

```diff
@@ -5056,7 +5056,7 @@ const item_category &item::get_category() const
     }
 
     static item_category null_category;
-    return type ? *type->category : null_category;
+    return type->category ? *type->category : null_category;
```

Comment:
This creates empty category:
![capture](https://cloud.githubusercontent.com/assets/5421954/19420307/65c467a6-93f1-11e6-9010-a830a230ca39.PNG)

Can we set 'OTHER' instead of null?